tgui/packages/tgui/interfaces/MedalsViewer.tsx

```diff
@@ -0,0 +1,60 @@
+import { BooleanLike, classes } from 'common/react';
+import { useBackend } from '../backend';
+import { Section } from '../components';
+import { Window } from '../layouts';
+
+interface MedalProps {
+  medals: Medal[];
+}
+
+interface Medal {
+  round_id: string;
+  medal_type?: string;
+  medal_icon?: string;
+  xeno_medal?: BooleanLike;
+  recipient_name?: string;
+  recipient_role?: string;
+  giver_name?: string;
+  citation?: string;
+}
+
+export const MedalsViewer = (props, context) => {
+  const { data, act } = useBackend<MedalProps>(context);
+  const { medals } = data;
+
+  return (
+    <Window width={700} height={350}>
+      <Window.Content scrollable className={'MedalsViewer'}>
+        {medals.map((medal) => {
+          const medalType = medal.medal_type
+            ? medal.medal_type
+            : 'Unknown Medal';
+          const sectionTitle = `Round ${medal.round_id} - ${medalType}`;
+          const sectionType = medal.xeno_medal ? 'xeno-medal' : 'human-medal';
+          return (
+            <Section
+              key={medal.citation}
+              title={sectionTitle}
+              className={sectionType}>
+              Issued to{' '}
+              <b>
+                {medal.recipient_name} ({medal.recipient_role})
```

Comment:
At the moment it appears unnecessary to display `{medal.recipient_role}` for xenos since their name will have it. Not sure any situation comes to mind where the name won't have the role.
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/f6cc9590-d865-41d0-a108-899ab0110dbd)